pyproject.toml

```diff
@@ -198,3 +206,19 @@ ignore-words-list = "objekt"
 ignore = [
   "W002",
 ]
+
+[[tool.mypy.overrides]]
+module = [
+  "nipype.*",
+  "seaborn",
+  "nilearn.*",
+  "templateflow.*",
+  "bids.*",
+  "svgutils",
+  "svgutils.*",
```

Comment:
Are these two entries necessary: https://github.com/nipreps/nireports/pull/146/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R217-R218

Wondering why `seaborn` and `pylab` do not require the `.*`: is their structure different that they do not require applying the override to their submodules or they way they are imported in the code typically (e.g. `import seaborn as sns`) makes these unnecessary?

Reply:
I don't know. I just noticed that `nipype` wasn't sufficient to block all nipype complaints, so I extended that without much trial-and-error to all the other failing imports.